user-management/.gitignore

```diff
@@ -0,0 +1,24 @@
+######################
+# Project Specific
+######################
+/build/www/**
```

Comment:
If you already exclude the build folder, you don't need to exclude this one as well. This is redundant.